### PR TITLE
chore: add volunteer Schleidens

### PR DESCRIPTION
**Pull request by Schleidens**

### CODE REVIEW

File: data/users/Schleidens.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Schleidens Alectine",
+  "bio": "Frontend Developer",
+  "image": "https://avatars.githubusercontent.com/u/53914451?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Vue", "Nuxt", "React", "Gatsby", "Next"]
```
Comment (codewithvoid) [inline data/users/Schleidens.json:8]: Can you keep this to max 2-3

Comment (codewithvoid) [inline data/users/Schleidens.json:8]: nit: add JS prefix. E.g VueJS, NextJS, etc

**Comment by codewithvoid** [comment]
Thank you for volunttering @Schleidens. Added some comments.

**Comment by Schleidens** [comment]
> Thank you for volunttering @Schleidens. Added some comments.

I'll clean it, thx for your review.

**Comment by codewithvoid** [comment]
Thanks for making quick changes!

**Pull request merged**